test/feature.spec.js

```diff
@@ -14,14 +14,63 @@ describe("zombie suite", function() {
     });
   });
 
-
   after(function(done) {
     this.server.close(done);
   });
 
-  it('finds a canvas on the homepage', function(done) {
+  it('has a mind mapp header', function(done) {
     var browser = this.browser;
     browser.assert.text('h1', 'mind mapp');
     done();
   });
+
+  //text input rather than text area
+  it('has a text area', function(done) {
+    var browser = this.browser;
+    browser.assert.element('textarea');
+    done();
+  });
+
+  it('has a canvas', function(done) {
+    var browser = this.browser;
+    browser.assert.element('canvas');
+    done();
+  });
+
+  it('has a rectangle button that can be clicked on', function(done) {
+    var browser = this.browser;
+    browser.pressButton('Rectangle').then(function() {
+    assert.ok(browser.success);
+    }).then(done, done);
+  });
+
+  it('has a circle button that can be clicked on', function(done) {
+    var browser = this.browser;
+    browser.pressButton('Circle').then(function() {
+    assert.ok(browser.success);
+    }).then(done, done);
+  });
+
+  //test background colour toggles tests are currently skipped- these need to be fixed to work
```

Comment:
Anything I can do with the css/html to help with that? Or is this an issue with Zombie?